lib/ansible/config/base.yml

```diff
@@ -1474,18 +1473,20 @@ INTERPRETER_PYTHON_DISTRO_MAP:
 INTERPRETER_PYTHON_FALLBACK:
   name: Ordered list of Python interpreters to check for in discovery
   default:
-  - /usr/bin/python
   - python3.9
```

Comment:
sounds like the consensus was to add 3.10 as well, since it'll be released by the time core 2.12 ships

Reply:
Sure thing.